### DISCUSSION TITLE
Importing notes from Google Keep

### DISCUSSION THREAD

**Discussion by Eve1374** [quillpad/quillpad#247] [Show and tell]
Hello,
In my search of a replacement for google Keep I found Quillpad, which has backup and restore features.
I made a quite rudimentary python script to convert google keep backups to Quillpad.

For me it worked for importing note titles, content, and pinned or not.

It import notes in the alphabetical order of google Keep backup file name (which is the note title if you put one), not in the order in which the notes are displayed in google keep. It doesn't keep text format and maybe if you have stuff that may be interpreted as markdown in your notes, it will be interpreted. it doesn't import the notes label and creation date. Not yet.

Steps to use :

-export your google keep notes with takeout
-unzip the downloaded archive
-move all json to a separate folder
-from this json folder run 
-python3 GKeepToQuillad.py *.json
-then run the sed command that is printed on the terminal
-compress the res/backup.json file in a zip
-import this zip in Quillpad (you may use KDE connect for example to send it to your phone) and hope it works

Here it is : [GKeepToQuillpad](https://github.com/Eve1374/GKeepToQuillpad)

**Comment by l0f3n** [comment]
Hey! I just wanted to promote the script i just created for the same purpose. It supports slightly more things than your original one https://github.com/l0f3n/gk2qp

**Comment by lriaudel** [comment]
It is possible to add it in quillpad app ? Many of people d'ont know how to lauch a python script.

**Reply by l0f3n** [reply]
You would have to talk to the original Quillpad developers for that. Would it help if it was a website where you could just upload your export and then download the converted file instead of having to run a python script?
